### PR TITLE
Patients table - new features

### PR DESCRIPTION
**Pull request by Johngtka**
ezgif com-video-to-gif

### CODE REVIEW

**Comment by Johngtka** [comment]
S & M commit mesage:
`feat: Actions buttons to handle visits for patients (#22)`

File: src/app/patient-table/patient-table.component.html [hunk 1]
```diff
@@ -45,6 +41,23 @@
...
         <td mat-cell *matCellDef="let element">{{ element.dob | age }}</td>
     </ng-container>
+    <ng-container matColumnDef="actions">
+        <th mat-header-cell *matHeaderCellDef>
+            {{ 'PATIENT_TABLE.ACTIONS' | translate }}
+        </th>
+        <td *matCellDef="let element">
+            <button
+                mat-icon-button
+                matTooltip="Add Visit"
```
Comment (macrog) [inline src/app/patient-table/patient-table.component.html:51]: tooltip should be 'new appointment' and it has to be translated

File: src/app/patient-table/patient-table.component.html [hunk 2]
```diff
@@ -45,6 +41,23 @@
...
+                (click)="startNewVisit(element)"
+            >
+                <mat-icon>add_circle_outline</mat-icon>
+            </button>
+            <button mat-icon-button matTooltip="Show history" [disabled]="true">
```
Comment (macrog) [inline src/app/patient-table/patient-table.component.html:56]: tooltip should be 'previous appointment' and it should be translated

**Review by macrog — CHANGES_REQUESTED** [review]
just a little change

File: src/app/patient-table/patient-table.component.html (hunk 1 above)
Comment (macrog) [reply src/app/patient-table/patient-table.component.html:51]: add as well matTooltipShowDelay=1500

File: src/app/patient-table/patient-table.component.html (hunk 2 above)
Comment (macrog) [reply src/app/patient-table/patient-table.component.html:56]: add as well `matTooltipShowDelay=1500`

**Review by macrog — APPROVED** [review]
good work! 🥇

**Pull request merged**